src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -30,6 +30,10 @@ const Body = BaseForm.extend({
     this.stopPolling();
   },
 
+  render () {
```

Comment:
why need this override?

Reply:
No need. Forgot to remove the debugging code :p